pkgs/servers/home-assistant/default.nix

```diff
@@ -331,6 +331,7 @@ in with py.pkgs; buildPythonApplication rec {
     voluptuous
     voluptuous-serialize
     yarl
+    ifaddr
```

Comment:
should come after `httpx`

Reply:
#152508 has this in the right place FWIW!